Example/Tests/CallSpec.swift

```diff
@@ -68,6 +68,70 @@ class CallSpec: QuickSpec {
             }
 
         }
+        
+        describe("Call with parameters") {
```

Comment:
💣 Wat te veel dubbele code:

```swift
let params = Parameters(type: Parameters.ParameterType.httpHeader, parameters: ["Accept-Language" : "en-US",
 +                                                                                                "Accept-Charset" : "utf-8"])
 +                let call = Call(path: "path", method: .GET, rootNode: nil, parameters: params)
 +                let request = call.request(withConfiguration: configuration)
```
Is bijna overal hetzelfde en je hebt enkel de request nodig. Maak dan een functie in de test die parameterType en parameters als parameter heeft.